site.ts

```diff
@@ -221,18 +272,95 @@ export function inject({ config, posthog }) {
         })
     }
 
+    const createRatingsPopup = (survey) => {
+        const scale = survey.questions[0].scale
+        const displayType = survey.questions[0].display
+        let ratingOptionsElement
+        if (displayType === 'number') {
+            ratingOptionsElement = document.createElement('div')
+            ratingOptionsElement.className = 'rating-options-buttons'
```

Comment:
you could pull these two lines out of the two sides of the if condition since they only vary by `displayType` 

but maybe that's nit-picky

Reply:
will do! 😃